src/libraries/System.Private.CoreLib/src/System/Double.cs

```diff
@@ -547,6 +547,9 @@ object IConvertible.ToType(Type type, IFormatProvider? provider)
         // IBinaryNumber
         //
 
+        /// <inheritdoc cref="IBinaryNumber{TSelf}.AllBitsSet" />
+        static double IBinaryNumber<double>.AllBitsSet => BitConverter.UInt64BitsToDouble(0xFFFFFFFFFFFFFFFF);
```

Comment:
Adding the spacers makes it slightly more readable. I personally tend to put them in groups of `4`
```suggestion
        static double IBinaryNumber<double>.AllBitsSet => BitConverter.UInt64BitsToDouble(0xFFFF_FFFF_FFFF_FFFF);
```